Shared/UI/Face Detection/FaceImageCaptureViewController.swift

```diff
@@ -222,23 +230,56 @@ extension FaceImageCaptureViewController: AVCaptureVideoDataOutputSampleBufferDe
         }
     }
 
-    private func startAssetWriter(with sampleBuffer: CMSampleBuffer) {
+    /// Makes the image black and white, and makes the background clear.
+    func blend(original framePixelBuffer: CVPixelBuffer,
```

Comment:
Just moved this.

Reply:
was just gonna ask if the background was clear or not.